security/tasks.py

```diff
@@ -479,12 +480,17 @@ def apply(self, args=None, kwargs=None, related_objects=None, **options):
 
     def apply_async(self, args=None, kwargs=None, related_objects=None, **options):
```

Comment:
cool this realy works? Where you find it?

Reply:
Yup.

I found some older questions on Stackoverflow. One possible solution was using custom Celery loader with hook for task initialization. Luckily some guy mentioned `django.db.close_old_connections` and it worked on first try. 🙂 Sadly these functions around database connection are poorly documented in Django and you really have to dig deep to find anything useful.